notebooks/002-openvino-api/002-openvino-api.ipynb

```diff
@@ -240,6 +240,45 @@
     "serialize(model_tf, xml_path=\"model/exported_tf_model.xml\")"
    ]
   },
+  {
+   "attachments": {},
+   "cell_type": "markdown",
+   "id": "8e7ba1b1",
+   "metadata": {},
+   "source": [
+    "### TensorFlow Lite Model\n",
+    "\n",
+    "[TFLite](https://www.tensorflow.org/lite) models saved for inference can also be passed to OpenVINO Runtime. Pass the filename with extension to `read_model` and exported an OpenVINO IR with `serialize`"
```

Comment:
which extension?

This sentence looks unfinished

Reply:
add `.tflite`